README.md

```diff
@@ -70,6 +70,30 @@ Then configure the rules you want to use within `rules` property of your `.eslin
 }
 ```
 
+### Run the plugin only against test files
+
+With the default setup mentioned before `eslint-plugin-testing-library` will be run against your whole codebase. If want to run this plugin only against your tests files, you can do that by using [ESLint `overrides`](https://eslint.org/docs/user-guide/configuring/configuration-files#configuration-based-on-glob-patterns).
+
+Assuming you are using the same pattern for your test files as [Jest by default](https://jestjs.io/docs/configuration#testmatch-arraystring), the following config would run `eslint-plugin-testing-library` only against your test files:
+
+```javascript
+// .eslintrc
+{
```

Comment:
```suggestion
module.exports = {
```

Reply:
Actually, I'm gonna rewrite it as json for consistency with the rest of the examples here.